Rakefile

```diff
@@ -4,5 +4,15 @@ rescue LoadError
   puts 'You must `gem install bundler` and `bundle install` to run rake tasks'
 end
```

Comment:
Frankly you can get rid if this begin/rescue here.

Basically, can you just steal the Rakefile from manageiq-content directly?  I can't see why they would be any different.